README.md

```diff
@@ -67,20 +70,21 @@ more.
 # Other Dependencies usages
 
 - [LiteCommands (by Rollczi ❤️)](https://github.com/Rollczi/LiteCommands)
-- [Lombok (Only for Configs)](https://projectlombok.org/)
+- [Lombok](https://projectlombok.org/)
 - [PaperLib](https://github.com/PaperMC/PaperLib)
-- [Spigot-API](https://www.spigotmc.org/wiki/spigot-gradle/)
+- [Spigot API](https://www.spigotmc.org/wiki/spigot-gradle/)
 - [Kyori Adventure](https://docs.adventure.kyori.net/)
-- [CDN](https://github.com/dzikoysk/cdn)
+- [cdn](https://github.com/dzikoysk/cdn)
```

Comment:
CDN to skrót (Configuration Data Notation)

Reply:
Faktycznie, sugerowałem się nazwą repozytorium na GitHubie.